### PR TITLE
Ansible fixes b0.72

### LINKED ISSUE #3406: Ansible roles on Galaxy need an update
Two problems:

- The `command` module complains about an unknown `warn` argument`.
- For v0.72, we need two pbench repos: a `pbench-<version>` repo for the agent RPM and the `pbench` repo for other stuff (`pbench-sysstat` in particular, but also potentially `screen`).

### PR DESCRIPTION
**Pull request by ndokos**
Fixes #3406 

- Fix an "unknown argument" error that the `command` module raises: it does not like the `warn` argument any longer, so delete the argument.
- Add the version-specific pbench repo to the list of repos that are added to the installation target(s).

### CODE REVIEW

File: agent/ansible/collection/roles/pbench_repo_install/defaults/main.yml [hunk 1]
```diff
@@ -3,11 +3,19 @@
 # provided by the `ndokos` COPR user.
 fedoraproject_username: ndokos
 pbench_repo_url_prefix: https://copr-be.cloud.fedoraproject.org/results/{{ fedoraproject_username }}
+pbench_aux_repo_name: pbench
 
 repos:
-  - name: "{{ pbench_repo_name }}"
-    user: "{{ fedoraproject_username }}"
+  - tag: "{{ pbench_repo_name }}"
+    user: "{{ fedoraproject_user }}"
```
Comment (webbnh) [inline agent/ansible/collection/roles/pbench_repo_install/defaults/main.yml:10]: Where is `fedoraproject_user` defined?  (I see a definition for `fedoraproject_username` at line 4....)

**Review by webbnh — APPROVED** [review]
Looks OK to me as far as I can tell (just so long as the change from `username` to `user` was intended...).

Regarding the failing tests....

- `TestUpload.test_bad_metadata_upload` seems to be failing because of a change to the time format (the test is looking for the time zone offset to be preceded by a `+` but the error from the Server has a space there);
- `TestServerAudit.test_get_status_start` and the other two failing tests are receiving half as many audit events as they are expecting.

But, I don't see how the changes in this PR prompt either of those failure modes...which suggests that something external is biting us again.

File: agent/ansible/collection/roles/pbench_repo_install/defaults/main.yml (hunk 1 above)
Comment (ndokos) [reply agent/ansible/collection/roles/pbench_repo_install/defaults/main.yml:10]: In my inventory file: I introduced an inconsistency and then propagated it back... Fix coming.

**Review by dbutenhof — APPROVED** [review]

**Review by webbnh — COMMENTED** [review]
👍

**Review by webbnh — APPROVED** [review]

**Pull request merged**